docs/README.md

```diff
@@ -5,33 +5,39 @@
 </p>
 
 # Monster Hunter Now Events
+## Table Of Contents
+* Calendar Subscription (Purpose)
+  - Steps to Subscribe Event Calendar <br>
+* Local Implementation of Calendar Subscription <br>
+* Tools <br>
+* References 
 
-This is a calendar subscription file for events in [Monster Hunter Now](https://monsterhunternow.com/).
+## Calendar Subscription
+This is a calendar subscription page for events in [Monster Hunter Now](https://monsterhunternow.com/).
```

Comment:
It's actually a calendar subscription file, not a page. The file generated is an ICS file that calendar applications can subscribe to via url.